packages/app/lib/devMenuItems.ts

```diff
@@ -9,27 +11,77 @@ if (__DEV__) {
   metroBundlerURL = scriptURL.split('index.bundle?')[0] ?? null;
 }
 
-const devMenuItems = [
-  {
-    name: 'Delete local database',
-    callback: () => purgeDb(),
-  },
+type ExpoDevMenuItem = Parameters<typeof registerDevMenuItems>[0][0];
+
+const simulatorOnlyMenuItems: ExpoDevMenuItem[] = [
   {
-    name: 'Drizzle studio (simulator only)',
+    name: 'Drizzle studio',
     callback: async () => {
       const path = getDbPath() ?? '';
       sendBundlerRequest('open-sqlite', { path });
     },
   },
   {
-    name: 'Dump sqlite database',
+    name: 'Dump SQLite',
     callback: async () => {
-      const path = getDbPath() ?? '';
-      sendBundlerRequest('dump-sqlite', { path });
```

Comment:
I didn't see any way that this was functional before, lmk if there's something I missed